### PR TITLE
Ability to change DEFAULT_SCENARIO_TYPE + some conf cleanups + stream…

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: nussknacker-dist/src/universal/conf/application.conf [hunk 1]
```diff
@@ -18,25 +19,50 @@ scenarioTypes {
...
 
 flinkDeploymentConfig {
-  jobManagerTimeout: 1m
   type: "flinkStreaming"
+  jobManagerTimeout: 1m
   restUrl: ${?FLINK_REST_URL}
   queryableStateProxyUrl: ${?FLINK_QUERYABLE_STATE_PROXY_URL}
 }
 
+streamingLiteDeploymentConfig {
```
Comment (mproch) [inline nussknacker-dist/src/universal/conf/application.conf:47]: Is there a reason to put it here?

Comment (arkadius) [reply nussknacker-dist/src/universal/conf/application.conf:47]: Just to be consistent with `flinkDeploymentConfig`. Should I inline both? WDYT?

Comment (mproch) [reply nussknacker-dist/src/universal/conf/application.conf:47]: I'd inline both, there is some reason to extract it in dev conf, as we share config between many configs, but here it's just noise IMO

File: nussknacker-dist/src/universal/conf/application.conf [hunk 2]
```diff
@@ -18,25 +19,50 @@ scenarioTypes {
...
-          kafkaProperties {
-            "schema.registry.url": ${?SCHEMA_REGISTRY_URL}
-          }
-        }
+        config: ${kafkaConfig}
+      }
+    }
+  }
+  "streaming-lite-embedded": {
+    deploymentConfig: ${streamingLiteDeploymentConfig}
```
Comment (dswiecki) [inline nussknacker-dist/src/universal/conf/application.conf:27]: maybe streamingLiteDeploymentConfig -> embeddedDeploymentConfig

Comment (arkadius) [reply nussknacker-dist/src/universal/conf/application.conf:27]: good point

**Review by mproch — APPROVED** [review]
looks cool :+1:

**Pull request merged**